docs/product/ai-in-sentry/ai-code-review/index.mdx

```diff
@@ -26,6 +26,20 @@ To enable AI Code Review in your GitHub organization or on specific repositories
    - `Show Generative AI Features`
    - `Enable AI Code Review`
 
+### GitHub Permissions
+
+AI Code Review requires specific GitHub permissions to function properly. When you install or update the Sentry GitHub integration, you'll need to accept the following permissions:
+
+- **Pull Requests (Read & Write)**: To read PR content and write code review comments
+- **Checks (Read & Write)**: To create status checks that show the AI Code Review results on your PRs
+- **Commit Statuses (Read & Write)**: To post status checks to commits and integrate with branch protection rules
```

Comment:
I _think_ we don't need this one. We use [this](https://docs.github.com/en/rest/checks/runs?apiVersion=2022-11-28#update-a-check-run) endpoint and it lists "Checks" repository permissions (write)

Let's check other endpoints used:
* https://docs.github.com/en/rest/checks/runs?apiVersion=2022-11-28#create-a-check-run
* https://docs.github.com/en/rest/checks/runs?apiVersion=2022-11-28#list-check-runs-for-a-git-reference

Looks like they all are for the `checks:write`